src/editors/vscode/tasks.json

```diff
@@ -0,0 +1,145 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+    "inputs": [
+        {
+            "id": "unittest_filter",
+            "default": "",
+            "type": "promptString",
+            "description": "Provide a unit test name you'd like to run exclusively"
+        }
+    ],
+    "tasks": [
+        {
+            "label": "Configure",
+            "group": "build",
+            "type": "shell",
+            "command": "python3 ./configure.py --cc gcc --compiler-cache=ccache --without-documentation --debug-mode --build-targets=\"static,tests,bogo_shim\"",
+            "presentation": {
+                "reveal": "always",
+                "panel": "shared",
+                "close": true
+            }
+        },
+        {
+            "label": "Build All",
+            "group": "build",
+            "type": "shell",
+            "linux": {
+                "command": "make -j $(nproc)"
+            },
+            "osx": {
+                "command": "make -j $(sysctl -n hw.logicalcpu)"
+            },
+            "dependsOn": "Configure",
```

Comment:
Same for the other tasks depending on `Configure`:
Do we really want to enforce a new configure for each run of the build task?
I guess the build performance is still acceptable because of the use of ccache. But it would still prevent a user from just running a different configure and reuse the other tasks (without having to change the configure task).

Reply:
Fair point. I'll remove those dependencies.